lib/screens/HomeScreen.dart

```diff
@@ -28,8 +30,10 @@ class HomeScreenState extends State<HomeScreen> {
 
   @override
   Widget build(BuildContext context) {
+    TrovUser profile = Provider.of<ProfileManager>(context).profile;
+
     return StreamBuilder<QuerySnapshot>(
-          stream: CloudFirebaseHelper.getLocationsStream(ProfileManager().profile.friends),
+          stream: CloudFirebaseHelper.getLocationsStream(profile.friends),
```

Comment:
Instead of this call to `getLocationsStream` should be replaced with a call to stream in the profile manager and that should internally use `getLocationsStream` and stream data back here. 

Idea is to have the UI separate from any of the backend calls and so `CloudFirebaseHelper` should ideally never be imported in a UI file like `HomeScreen`. What are your thoughts on this? Let me know.

Reply:
I like you're thinking.  I did a search on CloudFirebaseHelper, and there is also a direct call in GroupsScreen that could be refactored to move to GroupsManager.  
I will make both of these changes.

Only thing I would suggest, perhaps adding a LocationsManager to manage the cloud calls to the Locations Table, and not mix this with Profile.  What do you think?

Are you ok using Manager to describe these classes.  I was thinking Provider might be a good name as well.

Should I go ahead and convert tThemeManager to a provider?